src/Document/CommentManager.php

```diff
@@ -25,10 +26,8 @@ class CommentManager extends BaseDocumentManager implements CommentManagerInterf
     /**
      * @param int $page
      * @param int $limit
-     *
-     * @return Pager
      */
-    public function getPager(array $criteria, $page, $limit = 10, array $sort = [])
+    public function getPager(array $criteria, $page, $limit = 10, array $sort = []): PagerInterface
```

Comment:
Can you drop support for `sonata-project/datagrid-bundle` < 3.0 here? It will be awesome to make this change complex.

Reply:
I personally have a preference for small PR, I can try to make the bump from 2.x to 3.x on another one, and probably opening an issue in the meantime

Reply:
You can use this:
https://github.com/sonata-project/dev-kit/issues/993